packages/common/src/transformers/BarChartDataTransformer.ts

```diff
@@ -0,0 +1,68 @@
+import { type ResultRow } from '../types/results';
+import { type BarChartConfig } from '../types/visualizations';
+import { SqlRunnerResultsTransformer } from './ResultTransformers';
+
+// TODO: Should this just be ChartDataTransformer?
+export class BarChartDataTransformer {
+    private transformer: SqlRunnerResultsTransformer;
+
+    constructor(args: { data: ResultRow[] }) {
+        this.transformer = new SqlRunnerResultsTransformer(args);
+    }
+
+    public getEchartsSpec(config: BarChartConfig) {
```

Comment:
```suggestion
    public getEchartsSpec(config: BarChartConfig | undefined) {
```

This could be `undefined`